dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestInjectorInterface.cpp

```diff
@@ -102,9 +99,9 @@ TEST(DataInterfaceTest, Injector) {
   ASSERT_TRUE(Ifce.hasQ());
   ASSERT_DOUBLE_EQ(Ifce.getQ(), 499.0);
 
-  ASSERT_TRUE(Ifce.hasP());  // true since Load is common to Ifce and IfceNC
-  ASSERT_TRUE(Ifce.hasQ());
-  ASSERT_DOUBLE_EQ(IfceNC.getP(), 0.0);  // But 0.0 here since is not initialy connected
+  DYN::InjectorInterfaceIIDM IfceNC(Inj, "Injector initialy not connected");
+  ASSERT_FALSE(IfceNC.getInitialConnected());
```

Comment:
The case !getInitialConnected and P/Q not defined is missing

Reply:
done